pkg/oc/cli/cmd/newapp.go

```diff
@@ -723,7 +723,8 @@ func handleError(err error, baseName, commandName, commandPath string, config *n
 		// this print serves as a header for the printing of the errorGroups, but
 		// only print it if we precede with classification errors, to help distinguish
 		// between the two
-		fmt.Fprintln(buf, "Errors occurred during resource creation:")
+		fmt.Fprintf(buf, "Errors occurred during %s processing of arguments:", commandName)
```

Comment:
can we see a sample of this new output?  i'm not clear on what values "commandName" takes on

Reply:
For a new-build invocation it would be:

```
Errors occurred during new-build processing of arguments:
```

where the dump of the errorGroup follows ...i.e. `error: ....`

substitute `new-app` for `new-build` if it comes from `oc new-app ...`